src/main/resources/logback.xml

```diff
@@ -0,0 +1,15 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
+            <layout class="ch.qos.logback.contrib.json.classic.JsonLayout">
+                <jsonFormatter class="ch.qos.logback.contrib.jackson.JacksonJsonFormatter"/>
+                <appendLineSeparator>true</appendLineSeparator>
+                <timestampFormat>yyyy-MM-dd' 'HH:mm:ss.SSS</timestampFormat>
```

Comment:
Good catch, cut past error. I've fixed the format.

However, there's a possible issue that the logback format with `Z` actually comes out as `+0000` timezone offset.

Whereas the Elastic Co. formatter for the Tomcat Juli logger comes out as `Z`.

Semantically equivalent and local ISO 8601 but not sure if that'll trip up fluentd.

/cc @andrew-pickin-epi

Reply:
It will upset fluentd. However only to the extent that we can't update the record as it appear in kibana etc to ms.
If you're happy with timestamps to the nearest sec than it's not an issue.